team-builder/src/components/TeamForm.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+
+const TeamForm = props => {
+  console.log("props", props);
+  const [team, setTeam] = useState({ name: "", email: "", role: "" });
+
+  const handleChanges = e => {
+    // console.log(team);
+
+    setTeam({ ...team, [e.target.name]: e.target.value });
+  };
+
+  const submitForm = e => {
+    e.preventDefault();
+    props.addNewTeam(team);
+    setTeam
+    ({ name: "", email: "", role: "" });
+  };
+
+  return (
+    <form onSubmit={submitForm}>
+      <label htmlFor="name">Name</label>
+      <input
+        id="name"
+        type="text"
+        // Add placeholders
+        name="name"
+        onChange={handleChanges}
+        value={team.name}
+      />
+      <br /> <br />
```

Comment:
Please use CSS to structure your forms rather then `<br />`. `<br />` should only be used in `<address>` or when CSS is not available.